### PR TITLE
Feature/test suite tp

### PR DESCRIPTION
**Pull request by AsherSpurr**
# Pull Request
Test suite
## Description
E2E testing for the webpage
## Changes Made
- Finish testing for the education pages
- Add testing for singular tea fetch and information display
## Testing
- Landing page -> children, links, content of 1st and last article
- Buttons and nav links
- First and last tea pages display + content -> Blends + White
- First and last flip cards + content -> inside Blends + White
- First and last educational pages display + content -> Black + Green
- Display tea pages from educational buttons
- Add and removal of favorites to fav page
- Empty fave page
- URL's
- 4xx status
## Checklist
- [x] The code follows the project's coding standards.
- [x] Unit tests have been added or updated to cover the changes.
- [ x Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [x] The changes have been tested locally and pass all relevant tests.
- [x] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

### CODE REVIEW

**Comment by AsherSpurr** [comment]
Rock on!

**Comment by tednaphil** [comment]
Resolved the merge conflicts and updated failing tests to pass by updating selectors and setting the `cy.viewport()` to 1117 width per the media queries.

File: cypress/e2e/dashboard_spec.cy.ts [hunk 1]
```diff
@@ -4,96 +4,167 @@ describe('Spilled', () => {
...
+      .get('.card-cont').first().contains('p', 'sweet, vegetal, and delicate')
+      .get('.nav-favorites').click()
+      .url().should('include', 'favorites')
+      .get('p').contains('don\'t have any favorites')
+      .get('a').contains('Go Home')
   })
-  it('Displays tea info pages', () => {
-  
 
+  it('Displays tea info pages', () => {
```
Comment (tednaphil) [inline cypress/e2e/dashboard_spec.cy.ts:69]: Fabulous!

**Review by EthanDuvall — APPROVED** [review]
Nice tests!

**Review by corysanders3 — APPROVED** [review]
Nice changes. Glad we got those tests working.

File: cypress/e2e/dashboard_spec.cy.ts [hunk 2]
```diff
@@ -4,96 +4,167 @@ describe('Spilled', () => {
...
-    .get('nav').contains('a', 'Black').click()
-    .get('p').contains('Failed to fetch tea data')
-    //test remaining error page content
+      .visit('http://localhost:3000/')
+      .get('nav').contains('a', 'Black').click()
+      .get('p').contains('Failed to fetch tea data')
   })
+
   it('Displays error message if visiting a bad path', () => {
-    // cy.intercept('GET', 'https://boonakitea.cyclic.app/api/all', {
```
Comment (tednaphil) [inline cypress/e2e/dashboard_spec.cy.ts:91]: This wasn't necessary for visiting the bad path, so I removed the interception

**Review by tednaphil — APPROVED** [review]

**Pull request merged**